pyOCD/rtos/__init__.py

```diff
@@ -18,9 +18,11 @@
 from .argon import ArgonThreadProvider
 from .freertos import FreeRTOSThreadProvider
 from .zephyr import ZephyrThreadProvider
+from .rtx import RTXThreadProvider
 
 RTOS = {
           'Argon' : ArgonThreadProvider,
           'FreeRTOS' : FreeRTOSThreadProvider,
           'Zephyr' : ZephyrThreadProvider,
+          'RTX' : RTXThreadProvider,
```

Comment:
Let's make it explicitly RTX5, in case support for earlier RTX versions is added at some point. Same applies to the `rtx` module name.